docs/contribute_to_doc_guide.rst

```diff
@@ -0,0 +1,26 @@
+===================================
+Contributing to the documentation
+===================================
+This is to provide some guidance on contributing to setuptools'
+documentation for those unfamiliar with sphinx. setuptools' documentation is built using 
+`Sphinx <https://pypi.org/project/Sphinx/>`_ and written in `reStructuredText <https://docutils.sourceforge.io/rst.html>`_.
```

Comment:
It's probably better to link Sphinx's doc rather than PyPI page. Also, I'd probably set up intersphinx for this too.